control/available_plugin.go

```diff
@@ -51,12 +53,11 @@ const (
 )
 
 var (
-	// ErrPoolNotFound - error message when the plugin pool not found
```

Comment:
I felt they added more clutter than value.  

@geauxvirtual: do you feel they should be put back?